src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -442,31 +450,100 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
             return null;
         }
 
+        private bool IsParamsArrayExpanded(SemanticModel semanticModel, SyntaxNode node, SymbolInfo symbolInfo, CancellationToken cancellationToken)
+        {
+            if (symbolInfo.Symbol == null)
+            {
+                return false;
+            }
+
+            int argumentCount;
+            bool lastArgumentHasNameColon;
+            ExpressionSyntax lastArgumentExpression;
+
+            if (node is AttributeSyntax attribute)
+            {
+                argumentCount = attribute.ArgumentList.Arguments.Count;
+                lastArgumentHasNameColon = attribute.ArgumentList.Arguments.LastOrDefault()?.NameColon != null ||
+                    attribute.ArgumentList.Arguments.LastOrDefault()?.NameEquals != null;
+                lastArgumentExpression = attribute.ArgumentList.Arguments.LastOrDefault()?.Expression;
```

Comment:
would be good to NRT these files.

Reply:
I NRT'd this file. The Abstract one already was.